tests/ashley/test_forum_search.py

```diff
@@ -730,3 +731,80 @@ def test_forum_search_paginated_links(self):
             )
 
         self.assertNotContains(response, "?q=a5g3&amp;page=7")
+
+    def test_forum_search_show_direct_post_with_right_pagination(self):
+        """
+        Check that the link of the results targets directly the post in
+        the topic. Using this link, the user get directed at the right
+        page of the topic the post is related to.
+        """
+
+        user = UserFactory()
+        post = PostFactory(text="azatofind")
+        PostFactory.create_batch(49, topic=post.topic)
+        # Create a specific post between two batchs
+        post_fourth_page = PostFactory(text="azatofind", topic=post.topic)
+        PostFactory.create_batch(50, topic=post.topic)
```

Comment:
this will slow down your tests considerably. You could avoid that if you could temporarily set pagination to 2 items per page for example?